handlers_manager.go

```diff
@@ -155,7 +155,11 @@ func (hm *HandlersManager) RegisterHandler(h IHandler) error {
 
 		hm.handlerVersions[version.Route] = version
 
-		_, version.UseCache = handlerType.MethodByName(handlerMethodPrefix + "UseCache")
+		_, deprecatedUseCache := handlerType.MethodByName(handlerMethodPrefix + "UseCache")
+		_, deprecatedUseETag := handlerType.MethodByName(handlerMethodPrefix + "UseEtag")
+		if deprecatedUseCache || deprecatedUseETag {
+			return errors.New("UseCache and UseEtag marker methods no longer supported. Use http_json.DisableCache() and http_json.DisableETag()")
```

Comment:
It's information for developers using the library. I get it, I'll change it to panic so it was developer only specific message.